src/uct/api/v2/uct_v2.h

```diff
@@ -132,6 +132,15 @@ typedef enum {
     UCT_MD_MEM_DEREG_FIELD_ARG      = UCS_BIT(3)  /**< arg field */
 } uct_md_mem_dereg_field_mask_t;
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief field mask of @ref uct_iface_is_reachable_v2
+ */
+typedef enum {
+    UCT_IFACE_IS_REACHABLE_FIELD_DEV_ADDR   = UCS_BIT(0), /**< dev_addr field */
+    UCT_IFACE_IS_REACHABLE_FIELD_IFACE_ADDR = UCS_BIT(1), /**< iface_addr field */
+    UCT_IFACE_IS_REACHABLE_FIELD_ERROR_STR  = UCS_BIT(2), /**< error_str field */
```

Comment:
1. maybe rename error_str to info_str, since the function returns true/false and not error code
2. pls add flag for ERROR_STR_LEN

Reply:
fixed